changelog/freebsd-version.dd

```diff
@@ -0,0 +1,6 @@
+FreeBSD 10.3 is new minimum supported version
+
+This is because in order to support Elf common blocks, the linker
+ld 2.17.50 version or later is required. Although FreeBSD 9.2
+introduced ld 2.17.50, and may work, FreeBSD versions prior to 10.3
+are not officially supported.
```

Comment:
@jmdavis is this even remotely correct?

Reply:
Right now, the only version of FreeBSD we officially support is FreeBSD 11. Not even the FreeBSD people support anything older than FreeBSD 11 (11.2 specifically). The auto-tester only tests FreeBSD 11, and we dropped support for FreeBSD 10 some time ago. So, there should be no need for this changelog entry.